src/frontend/src/lib/components/hero/about/AboutHow.svelte

```diff
@@ -0,0 +1,16 @@
+<script lang="ts">
+	import AboutItem from '$lib/components/hero/about/AboutItem.svelte';
+	import { i18n } from '$lib/stores/i18n.store';
+	import { modalStore } from '$lib/stores/modal.store';
+
+	export let asMenuItem = false;
+	export let onClick: (() => void) | undefined = undefined;
+</script>
+
+<AboutItem
+	labelAsTitle={$i18n.about.how.text.title}
```

Comment:
can we use slot to pass text?

Reply:
check last version, is that what you meant?